lib/membrane_rtc_engine/endpoints/hls_endpoint.ex

```diff
@@ -294,26 +297,50 @@ if Enum.all?(
         manifest_module: Membrane.HTTPAdaptiveStream.HLS,
         target_window_duration: state.target_window_duration,
         target_segment_duration: state.target_segment_duration,
+        muxer_segment_duration: state.target_segment_duration - Membrane.Time.seconds(1),
         persist?: false,
         storage: %Membrane.HTTPAdaptiveStream.Storages.FileStorage{
           directory: directory
         },
         hls_mode: state.hls_mode
       }
 
+      audio_mixer = %Membrane.AudioMixer{
+        caps: %Membrane.RawAudio{
+          channels: 1,
+          sample_rate: 48_000,
```

Comment:
In general this should be taken from the track struct.

Reply:
We can take `sample_rate` from the track but not channels and sample_format, so I will add audio_mixer_configuration the same as it is for the compositor.